stage_descriptions/geospatial_3_ck3.md

```diff
@@ -0,0 +1,50 @@
+In this stage, you'll add support for validating the latitude and longitude provided in the `GEOADD` command.
+
+### The `GEOADD` command (Validating Coordinates)
+The latitude and longitudes used in the GEOADD command should be in a certain range. Valid longitudes are from -180 to 180 degrees. Valid latitudes are from -85.05112878 to 85.05112878 degrees. Both of these limits are inclusive.
+Example use case:
+
+```
+# Invalid latitude
+> GEOADD places 180 90 test1
+(error) ERR invalid longitude,latitude pair 180.000000,90.000000
+
+# Invalid longitude
+> GEOADD places 181 0.3 test2
+(error) ERR invalid longitude,latitude pair 181.000000,0.300000
+
+# Roundinf off to 6 digits in case of errors
+> GEOADD loc 122 90.23454393 test3
+(error) ERR invalid longitude,latitude pair 122.000000,90.234544
+
+```
+
+In the response, both latitude and longitude values are rounded to 6 digits after the decimal point.
```

Comment:
Let's not be this strict - just assert that the message starts with `invalid`. Also less for us to explain re: rounding.